.goreleaser.yml

```diff
@@ -33,12 +33,42 @@ furies:
     secret_name: "GEMFURY_PUSH_TOKEN"
     skip: "{{ gt (len .Prerelease) 0 }}"
 brews:
-  - repository:
+  - name: "{{.ProjectName}}"
```

Comment:
this is the default, you can leave it out if you want to

Reply:
I think this is useful just to be explicit that this is the default latest Formula.